projects/components/src/dropdown/dropdown.component.ts

```diff
@@ -307,10 +311,28 @@ export class DropdownComponent implements AfterViewInit {
      * @param event Space or Enter key press event
      */
     onDropdownItemActivated(event: Event): void {
-        if (!event) {
+        event.stopPropagation();
+        event.target.dispatchEvent(new Event('click'));
```

Comment:
a little bit of comment why we need to dispatch a click event rather than calling  onItemClicked(item)